controllers/apply-next/simple/form-model.js

```diff
@@ -9,6 +9,12 @@ const sectionProject = {
         en: 'Your Project',
         cy: '(WELSH) Your Project'
     },
+    introduction: {
```

Comment:
This field is for the summary screen, eg:

![image](https://user-images.githubusercontent.com/394376/53090244-b4bef480-3506-11e9-9fc3-36b5ab0971a2.png)